bin/fuel-convert

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+import argparse
+import importlib
+import pkgutil
+
+from fuel import converters
+
+
+if __name__ == "__main__":
```

Comment:
I think what we will want to do in the long term is add [sub-commands](https://docs.python.org/3/library/argparse.html#sub-commands) for each dataset that lets you control options related to that dataset specifically.